src/utilities/getAppGuiPath.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { getAppGuiPath } from './getAppGuiPath'
+
+describe('getAppGuiPath', () => {
```

Comment:
this looks like we are testing this twice

Reply:
Fixed. See https://github.com/kumahq/kuma-gui/pull/445#discussion_r1017696184.